tests/test_contrib/test_missing/test_dispersion.py

```diff
@@ -41,20 +41,25 @@ def setUp(self):
         if os.name == 'nt': # Windows
             self.tol = 5.0
 
+    @pytest.mark.skipif(pd is None, reason="test requires pandas")
```

Comment:
Same question as before - why does this test require pandas? Sorry if I'm missing something obvious!